src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HiddenField/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="HiddenField"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="neverSuppressedChecks" value="HiddenFieldCheck" />
```

Comment:
Please remove it and update the above patch to have changes in this line as well `final int test = 1;  // violation without filter` so only one violation is expected.

Reply:
@rdiachenko I try it in context/case2 and it cause a error:
```
Expected :/Users/hgh/Documents/GitHub/patch-filters/src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HiddenField/context/case2/Test.java:6:30: 'test' hides a field.
Actual   :Audit done.
```

Reply:
1. Of course there is no violation because you fixed it by your change in the patch :) Please do this change in the patch so the violation occurs:
```
-        final int test1 = 1;
+        final int test = 1;
```
2. Why do you use `supportContextStrategyChecks` in the case2? Can you remove it and use this config:
```
        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
            <property name="file" value="${tp}/defaultContext.patch" />
            <property name="strategy" value="context" />
        </module>
```

Reply:
2. after https://github.com/checkstyle/patch-filters/issues/218, we have removed all hardcoded checks in filter, so every check that using `context` strategy need use `supportContextStrategyChecks`.